src/neo/SmartContract/ApplicationEngine.Contract.cs

```diff
@@ -118,6 +119,16 @@ protected internal void DestroyContract()
                     Snapshot.Storages.Delete(key);
         }
 
+        internal bool ContractExists(UInt160 contractHash, string method)
+        {
+            if (contractHash is null) return false;
+
+            ContractState contract = Snapshot.Contracts.TryGet(contractHash);
+            if (contract is null) return false;
+
+            return method == null || contract.Manifest.Abi.GetMethod(method) != null;
```

Comment:
```suggestion
            return method != null && contract.Manifest.Abi.GetMethod(method) != null;
```

Reply:
If you want to check if the contract exists, you can use `null`

Reply:
Done